### PR TITLE
fix: make mux-player size based on video element

### PR DESCRIPTION
**Pull request by luwes**
~Requires this PR in Media Chrome: https://github.com/muxinc/media-chrome/pull/205~

Upgraded media-chrome to 0.6.0

### CODE REVIEW

File: packages/mux-player/src/styles.css [hunk 1]
```diff
@@ -21,9 +18,6 @@ media-controller {
   --media-control-hover-background: transparent;
   --media-range-track-background-color: rgba(255, 255, 255, 0.5);
   --media-range-track-border-radius: 3px;
-  --media-aspect-ratio: auto;
-  display: block;
-  width: 100%;
```
Comment (gkatsev) [inline packages/mux-player/src/styles.css:26]: width is still needed as well

Comment (gkatsev) [inline packages/mux-player/src/styles.css:26]: width is still needed as well

**Review by gkatsev — CHANGES_REQUESTED** [review]

**Review by gkatsev — APPROVED** [review]

File: packages/mux-player/src/styles.css (hunk 1 above)
Comment (luwes) [reply packages/mux-player/src/styles.css:26]: added, I didn't reproduce anything that required it yet or broke without it but it does make sense to have `width: 100%` as well

Comment (gkatsev) [reply packages/mux-player/src/styles.css:26]: I was seeing some issues particularly with portrait videos and trying to set the width, which this resolved.

**Pull request merged**